doc/architecture/decisions/0005-data-retention-policy.md

```diff
@@ -0,0 +1,110 @@
+# 5. Data Retention Policy
+
+Date: 2022-12-20
+
+## Status
+
+Proposed
+
+## Context
+
+We have agreed that we need to implement a data retention policy, and the basics of this has been created in [#1325](https://github.com/theopensystemslab/planx-new/pull/1325).
+
+As part of this, it surfaced that we do not have an agreed and formalised data retention policy which lives outside of the code to serve as a reference.
+
+## Decision
+Here are diagrams illustrating our current and proposed data retention policies - 
+
+> Please note that any events hitting the terminal line indicate they are held indefinately
+
+```mermaid
+gantt
+    title Current Policy
+    dateFormat  YYYY-MM-DD
+
+    section User Journey
+    User completes application: uj1, 2022-01-01, 28d
+
+    section User Data
+    Lowcal Session: 2022-01-01, 35d
+    Uploaded files: 2022-01-14, 2022-12-31
+    Session backups: after uj1, 2022-12-31
+
+    section Audit Logs
+    Submission audit logs: after uj1, 2022-12-31
+    Reconciliation audit logs: 2022-01-14, 2022-12-31
+
+    section Event Metadata
+    Submission Event Metadata: after uj1, 2022-12-31
+    Reconciliation Event Metadata: 2022-01-14, 2022-12-31
+    
+    section Feedback
+    FeedbackFish (last 250 records, rolling): 2022-01-14, 2022-12-31
+    Feedback Learning Log (anonymised): 2022-01-14, 2022-12-31
+```
+
+```mermaid
+gantt
+    title Proposed Policy
+    dateFormat  YYYY-MM-DD
+
+    section User Journey
+    User completes application: uj1, 2022-01-01, 28d
+
+    section User Data
+    Lowcal Session: 2022-01-01, 210d
+    Lowcal Session (Sanitised): 2022-12-31
+    Uploaded files: 2022-01-14, 182d
+    Session backups: after uj1, 182d
+    Session backups (Sanitised): 2022-12-31
+
+    section Audit Logs
+    Submission audit logs: after uj1, 182d
+    Submission audit logs (Sanitised): 2022-12-31
+    Reconciliation audit logs: 2022-01-14, 182d
+    Reconciliation audit logs (Sanitised): 2022-12-31
+
+    section Event Metadata
+    Submission Event Metadata: after uj1, 182d
+    Reconciliation Event Metadata: 2022-01-14, 182d
+
+    section Feedback
+    FeedbackFish (last 250 records, rolling): 2022-01-14, 2022-12-31
+    Feedback Learning Log (anonymised): 2022-01-14, 2022-12-31
+```
+
+Here are the broad catergory of changes - 
+
+|| Current Retention Period | Proposed Retention Period |
+|---|---|---|
+| **User Data**| Lowcal Sessions - 7 days, then deleted. Session Backups - Indefinite | 6 months from submission date, then sanitised and held indefinitely |
+| **Uploaded Files**| Indefinite | 6 months from upload date |
```

Comment:
There's a ticket to implement this here - https://trello.com/c/XRcoHTyW/2237-implement-agreed-data-retention-policy-for-user-uploaded-files